chrome/content/zotero/xpcom/utilities.js

```diff
@@ -455,11 +455,15 @@ Zotero.Utilities = {
 	 *   <ZOTEROBREAK/> => <br/>
 	 *   <ZOTEROHELLIP/> => &#8230;
 	 *
-	 * @param {String} str
+	 * @param {String|Number} str
 	 * @return {String}
 	 */
 	"htmlSpecialChars":function(str) {
-		if (str !== undefined && typeof str != 'string') {
+		if ((str || str == 0) && typeof str != 'string') {
```

Comment:
It could be extensions, but also [this piece of code](https://github.com/zotero/zotero/blob/355e25d9a95df0fa810e82c23a266ef615e0e27b/chrome/content/zotero/xpcom/utilities.js#L1293-L1293), which technically shouldn't ever be broken, but `xpath()` is used by a lot of translators